package.json

```diff
@@ -1,36 +1,42 @@
 {
-    "name": "honeypot",
-    "version": "1.0.0",
-    "description": "",
-    "main": "app.js",
+    "name": "hash-honeypot",
+    "version": "1.0.2",
+    "description": "HTTP Agnostic Software Honeypot (HASH) - honeypot framework to build low interaction honeypots",
+    "main": "index.js",
+    "bin": {
+        "hash-honeypot": "bin/hash-honeypot"
+    },
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
         "prettier": "./node_modules/prettier/bin-prettier.js --write './**/*.js'",
         "prettier-check": "./node_modules/prettier/bin-prettier.js --check --ignore-unknown",
         "generate-third-party-licenses": "./node_modules/license-checker/bin/license-checker --csv --customPath .license-checker.json > LICENSE-3rdparty.csv"
     },
-    "author": "",
-    "license": "ISC",
+    "author": "Datadog Security Research team",
+    "license": "Apache-2",
     "dependencies": {
         "@faker-js/faker": "^7.6.0",
         "chalk": "^4.1.2",
         "clear": "^0.1.0",
         "clui": "^0.3.6",
+        "commander": "^11.0.0",
```

Comment:
looks like this makes https://github.com/DataDog/HASH/blob/main/LICENSE-3rdparty.csv outdated, can we autogenerate it?

Reply:
Updated it 👍🏻